charts/IndicatorDropdown.tsx

```diff
@@ -26,47 +76,256 @@ export class IndicatorDropdown extends React.Component<IndicatorDropdownProps> {
+                <div
+                    className={classnames({
+                        "indicator-browser": true,
+                        "is-open": this.props.isOpen
+                    })}
+                >
+                    <Select
+                        autoFocus={true}
+                        backspaceRemovesValue={false}
+                        escapeClearsValue={false}
+                        components={{
+                            Option: IndicatorOptionWrapper,
+                            DropdownIndicator: null,
+                            IndicatorSeparator: null
+                        }}
+                        controlShouldRenderValue={false}
+                        filterOption={() => true} // disable filtering, will be handled elsewhere
+                        getOptionValue={this.getValue}
+                        hideSelectedOptions={false}
+                        isClearable={false}
+                        isLoading={this.isLoading}
+                        menuIsOpen={true}
+                        onChange={this.onChange}
+                        onInputChange={this.onInputChange}
+                        options={this.filteredIndicators}
+                        placeholder="Search..."
+                        styles={selectStyles}
+                        tabSelectsValue={false}
+                        value={this.indicators.find(
+                            i => i.id === this.props.selectedId
+                        )}
+                    />
+                </div>
+                {this.props.isOpen && <Blanket onClick={this.onClose} />}
+            </React.Fragment>
         )
     }
 }
+
+const IndicatorOptionWrapper = (props: OptionProps<Indicator>) => {
+    const {
+        className,
+        data,
+        isDisabled,
+        isFocused,
+        isSelected,
+        innerRef,
+        innerProps
+    } = props
+    return (
+        <div
+            ref={innerRef}
+            className={classnames(
+                {
+                    "indicator-option": true,
+                    "indicator-option__disabled": isDisabled,
+                    "indicator-option__focused": isFocused,
+                    "indicator-option__selected": isSelected
+                },
+                className
+            )}
+            style={{ cursor: "pointer" }}
+            {...innerProps}
+        >
+            <IndicatorOption indicator={data as Indicator} />
+        </div>
+    )
+}
+
+const IndicatorOption = (props: { indicator: Indicator }) => {
+    const { indicator } = props
+    return (
+        <div className="indicator-list-item">
+            <h3 className="title">{indicator.title}</h3>
+            {indicator.subtitle && (
+                <p className="subtitle">{indicator.subtitle}</p>
+            )}
+            {indicator.sourceDesc && (
+                <p className="metadata">{indicator.sourceDesc}</p>
+            )}
+        </div>
+    )
+}
```

Comment:
Maybe you can "just" make use of `formatOptionLabel`, like I did [for the color scheme dropdown](https://github.com/owid/owid-grapher/blob/18d1a6d8eec9d4d28ce17efba66935120d4d593e/admin/client/ColorSchemeDropdown.tsx#L83-L113)?

The advantage is that you don't have to care about stuff like classes for focused, selected and stuff. The disadvantage is that you're probably more restricted.